Base/build.gradle

```diff
@@ -19,19 +19,6 @@ dependencies {
     testImplementation project(":base-test-utils")
 }
 
-sourceSets {
-    main {
-        resources {
-            srcDir 'src/main/java' exclude '**/*.java'
```

Comment:
There is only one non-.java file in src/main, and that is in src/main/resources already (Base.gwt.xml)